File: Modul-2/Soal-5/PRAK205-2410817210008-AhmadUlyani.py
import math
a, b = map(float, input().split())
c = math.sqrt(b * b - a * a)
print(f"Alas = {c:.0f} cm")
print(f"Tinggi = {a:.0f} cm")
keliling = a + b + c
print(f"Keliling = {keliling:.0f} cm")
luas = a * c / 2
print(f"Luas = {luas:.0f} cm^2")

print("")
a = float(input(""))
b = float(input(""))
c = math.sqrt(b * b - a * a)
print(f"Alas = {c:.0f} cm")
print(f"Tinggi = {a:.0f} cm")
keliling = a + b + c
print(f"Keliling = {keliling:.0f} cm")
luas = a * c / 2
print(f"Luas = {luas:.0f} cm^2")

File: Modul-2/Soal-5/PRAK205-2410817210008-AhmadUlyani.c
#include <stdio.h>
#include <math.h>

int main (){
    int a, b, c, keliling, luas;
    scanf("%d %d", &a, &b);
    c = sqrt(b * b - a * a);
    printf("Alas = %d cm\n", c);
    printf("Tinggi = %d cm\n", a);
    keliling = a + b + c;
    printf("Keliling = %d cm\n", keliling);
    luas = a * c / 2;
    printf("Luas = %d cm^2\n", luas);

    printf("\n");
    scanf("%d %d", &a, &b);
    c = sqrt(b * b - a * a);
    printf("Alas = %d cm\n", c);
    printf("Tinggi = %d cm\n", a);
    keliling = a + b + c;
    printf("Keliling = %d cm\n", keliling);
    luas = a * c / 2;
    printf("Luas = %d cm^2\n", luas);

    return 0;
}